changelog.d/5835.feature

```diff
@@ -0,0 +1 @@
+Ability to send registration emails from the homeserver.
```

Comment:
```suggestion
Add the ability to send registration emails from the homeserver rather than delegating to an Identity Server.
```

Reply:
FYI I think identity server is usually lowercase, whereas Identity Service API is uppercase.

Reply:
I think you're right!